inc/ocf_core.h

```diff
@@ -16,6 +16,31 @@
 #include "ocf_io.h"
 #include "ocf_mngt.h"
 
+struct ocf_core_info {
+	/** Core size in cache line size unit */
+	uint64_t core_size;
+
+	/** Core size in bytes unit */
+	uint64_t core_size_bytes;
+
+	struct {
```

Comment:
Why are you putting these two fields (flushed / dity) in an anonymous union?

Reply:
Those stats refers to ongoing flush operation.

Reply:
I added comment in code